app/services/diagnostics/move_inspector.rb

```diff
@@ -24,8 +24,9 @@ def generate
         move type:\t#{move.move_type}
         from location:\t#{move.from_location&.title}
         to location:\t#{move.to_location&.title}
+        supplier:\t#{move.supplier&.name}
```

Comment:
do we always have suppliers on moves? checking validations seems optional

Reply:
there are historic moves which don't have a supplier_id defined; and in the case of multiple suppliers to the same location I think there was a case when a move can be initialised (proposed) without a supplier and it updated later?